### DISCUSSION TITLE
Oxalis & EESPA Interoperability Network (EIN)

### DISCUSSION THREAD

**Discussion by OysteinLq** [OxalisCommunity/oxalis#540] [Ideas]
Hi, as you may know, there's a European trade association called EESPA who are working on what they call the EESPA Interoperability Network (EIN)

EIN seems to be very close to a Peppol as far as the underlying technology goes, using CEF, AS4, SML/SMP, UBL etc.  Has anyone considered whether this is something that Oxalis could or should be adapted to support? Or if indeed any adaptation will be required?

For those not familiar, I think a good starting point is on this page: https://eespa.eu/new-interoperability , under the "EESPA Interoperability Network (EIN) under development" banner.

I'm not sure how far off EIN is from being a viable service, so perhaps it's far too early to even consider the question, but I thought I'd ask in case anyone has thoughts on the matter.

**Comment by aaron-kumar** [comment]
Thanks for asking and being active player @OysteinLq . 

We in "The Oxalis Management Team" are regularly following and have close watch on global standards, framework and specifications from OpenPeppol, CEF, GIF, OASIS etc. and organizations & network like EESPA and BPC etc

Our immediate and main focus is to continuously adhere to latest OpenPeppol specification but that does not mean that these activities/organization/standards are out of our radar. At this moment, I can only say that those topics are part of our regular discussions.

**Comment by OysteinLq** [comment]
Thanks @aaron-kumar , it's good to hear these other networks are at least considered
